build.xml

```diff
@@ -360,24 +362,28 @@ Please make sure that:
     </target>
 
     <target name="-test-unix" unless="isWindows">
-    	<exec executable="sh" dir="test">
-    		<env key="JOLIE_HOME" file="dist/jolie" />
-    		<arg value="../dist/launchers/unix/jolie" />
-    		<arg value="test.ol" />
-    	</exec>
-    </target>
+			<exec executable="sh" dir="test">
+				<env key="JOLIE_HOME" file="dist/jolie" />
+				<arg value="../dist/launchers/unix/jolie" />
+				<arg value="-l" />
+				<arg value="extensions/private/WS-test/dist/WS-test.jar" />
+				<arg value="test.ol" />
+			</exec>
+		</target>
```

Comment:
This will break the ability of running simply `jolie test.ol`. Please put the library argument you need in the Jolie code (e.g., by using loadLibrary@Runtime or by embedding something and passing the `-l` option in the embedding string).

Reply:
Fixed.